src/PrestaShopBundle/Resources/config/routing/admin/improve/modules/_modules.yml

```diff
@@ -41,6 +44,17 @@ admin_module_manage:
         keyword:
         _controller: PrestaShopBundle:Admin/Improve/Module:manage
         _legacy_controller: AdminModulesManage
+        _legacy_link: AdminModulesManage
+
+admin_module_manage_alias:
```

Comment:
make `_legacy_link` a possible array for aliases will be better, no?
```yml
_legacy_link:
  - AdminModuleManager
  - AdminModulesSf
```

Reply:
yeah, I'm gonna add this feature

Reply:
done!